### PR TITLE
Add LinkKeyboard 

### PR DESCRIPTION
**Pull request by ayushgoel**

### CODE REVIEW

**Comment by ayushgoel** [comment]
The build is failing due to links not introduced by me 😢 ..

File: contents.json [hunk 1]
```diff
@@ -2522,6 +2522,18 @@
...
     {
+      "title": "Link Keyboard : My Links Everywhere",
+      "category-ids": [
+        "keyboard",
+        "extension"
+      ],
+      "description": "App to put all your links in one place and make them accessible via a keyboard.",
+      "source": "https://github.com/ayushgoel/LinkKeyboard",
+      "itunes": "https://itunes.apple.com/in/app/link-keyboard-my-links-everywhere/id1098798450",
+      "license": "MIT",
```
Comment (dkhamsing) [inline contents.json]: mit should be lower case

File: contents.json [hunk 2]
```diff
@@ -2522,6 +2522,18 @@
       ]
     },
     {
+      "title": "Link Keyboard : My Links Everywhere",
+      "category-ids": [
+        "keyboard",
+        "extension"
```
Comment (dkhamsing) [inline contents.json]: can you remove extension, even though it is correct, these types of apps are only keyboard right now

File: contents.json [hunk 3]
```diff
@@ -2522,6 +2522,18 @@
...
+      ],
+      "description": "App to put all your links in one place and make them accessible via a keyboard.",
+      "source": "https://github.com/ayushgoel/LinkKeyboard",
+      "itunes": "https://itunes.apple.com/in/app/link-keyboard-my-links-everywhere/id1098798450",
```
Comment (dkhamsing) [inline contents.json]: if this link also works without /in, please remove /in

File: contents.json [hunk 4]
```diff
@@ -2522,6 +2522,18 @@
       ]
     },
     {
+      "title": "Link Keyboard : My Links Everywhere",
+      "category-ids": [
+        "keyboard",
+        "extension"
+      ],
+      "description": "App to put all your links in one place and make them accessible via a keyboard.",
```
Comment (dkhamsing) [inline contents.json]: Please remove "app", should starts with "Put all your ..."

File: contents.json [hunk 5]
```diff
@@ -2522,6 +2522,18 @@
...
+      "source": "https://github.com/ayushgoel/LinkKeyboard",
+      "itunes": "https://itunes.apple.com/in/app/link-keyboard-my-links-everywhere/id1098798450",
+      "license": "MIT",
+      "suggested_by": "@ayushgoel"
+    },
```
Comment (dkhamsing) [inline contents.json]: Missing  
"date_added"  with today's date

Comment (dkhamsing) [inline contents.json]: missing "tags" = ["swift"]

**Comment by dkhamsing** [comment]
Don't worry about the build failures

**Comment by ayushgoel** [comment]
@dkhamsing Updated!

**Comment by dkhamsing** [comment]
Thx!

**Pull request merged**